oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/mongo/ReadPreferenceIT.java

```diff
@@ -116,10 +116,11 @@ public void testReadWriteMode() {
         mongoDS.setReadWriteMode(rwMode(ReadPreference.primary()));
         assertEquals(ReadPreference.primary(), mongoDS.getConfiguredReadPreference(NODES));
 
-        mongoDS.setReadWriteMode("readPreference=secondary&w=2&safe=true&j=true");
+        // Command failed with error 2 (BadValue): 'cannot use 'w' > 1 when a host is not replicated'
+        mongoDS.setReadWriteMode("readPreference=secondary&w=1&safe=true&j=true");
```

Comment:
I think this change doesn't affect in what the test wants to check.
Unless we start the cluster as a replica set, Mongo doesn't let you specify a 'w' value bigger than 1.